src/sidebar/components/test/sort-menu-test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const { createElement } = require('preact');
+const { shallow } = require('enzyme');
+
+const SortMenu = require('../sort-menu');
+const MenuItem = require('../menu-item');
+
+describe('SortMenu', () => {
+  let fakeState;
+  let fakeStore;
+
+  const createSortMenu = () => {
+    return shallow(<SortMenu />);
+  };
+
+  beforeEach(() => {
+    fakeState = {
+      sortKey: 'Location',
+      sortKeysAvailable: ['Newest', 'Oldest', 'Location'],
```

Comment:
When testing this I found what I think is an existing bug where `sortKeysAvailable` does not always correctly update to reflect the active tab, eg. if the tab is switched automatically rather than by the user clicking on the tab. It shouldn't really be a piece of state in the store but rather a value that is derived from the active tab.

I'll leave that as an exercise for the future.